src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -330,20 +333,33 @@ def get_attrs(
                 incident_map[incident.id] = serialize(incident, user=user)
 
         serialized_alert_rules = serialize(alert_rules, user=user)
-        rules = serialize(
+        serialized_map_by_id = {x.id: x for x in serialized_alert_rules}
+
+        serialized_issue_rules = serialize(
             [x for x in item_list if isinstance(x, Rule)],
             user=user,
             serializer=RuleSerializer(expand=self.expand),
         )
+        serialized_issue_rule_map_by_id = {x.id for x in serialized_issue_rules}
```

Comment:
Nit: just get the keys of the data structure from earlier
```suggestion
        serialized_issue_rule_ids = serialized_map_by_id.keys()
```
--
Do we need to create this variable or could we just use the map?

Reply:
Ah - the serialized alert rules are different than the serialized issue rules.

and yea - the map of id -> instance _is_ still needed to easily grab the instance